src/containers/GamesPage/index.jsx

```diff
@@ -1,12 +1,40 @@
 import React from "react";
+import { useTranslation } from "react-i18next";
 import GamesPageImageGrid from "../../components/gamespage/GamesPageImageGrid";
 import BottomCurve from "../../components/BottomCurve";
+import memoryGame from "../../images/gameScreenshots/memoryGame.svg";
+import puzzleGame from "../../images/gameScreenshots/puzzleGame.svg";
+import quizGame from "../../images/gameScreenshots/quizGame.svg";
+import matchingGame from "../../images/gameScreenshots/matchingGame.svg";
 
 const GamesPage = () => {
+  const [t] = useTranslation();
+  // agreed naming is as follows:
+  // image file name: memoryGame
+  // game title key in locales: memoryGame.title
+  // game route: memorygame
+  const gameScreenshots = [memoryGame, puzzleGame, matchingGame, quizGame];
+  const gamePostersObj = {};
+  // gameScreenshotDir example /static/media/memoryGame.84fe967a.svg
+  for (const gameScreenshotDir of gameScreenshots) {
+    const screenshotDirParts = gameScreenshotDir.split("/");
+    // example memoryGame.84fe967a.svg
+    const lastScreenshotDirPart =
+      screenshotDirParts[screenshotDirParts.length - 1];
+    // example memoryGame
```

Comment:
put a colon after example, here and everywhere else

Reply:
PTAL